pkg/client/client.go

```diff
@@ -564,8 +564,10 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 			eIP = "Enabled"
 		}
 
+		socketLB := "Disabled"
 		protocols := ""
 		if hs := sr.KubeProxyReplacement.Features.HostReachableServices; hs.Enabled {
+			socketLB = "Enabled"
 			protocols = strings.Join(hs.Protocols, ", ")
 		}
```

Comment:
But if you have kpr disabled, then `Protocols:` would not be shown as indicator, no?

Reply:
Yep, but it's not clear without reading the code / docs.

Reply:
But if status says 'strict', then it's indicator that it has to be enabled. If it says 'disabled', then it has to be disabled as well. So only in case of partial this questions comes up, and there we show the protocols. Maybe this is just a matter of documentation? What does the user make out of 'Socket LB'? If it's just for us debugging isn't the code as-is or sysdump sufficient?

Reply:
Should we just combine this with the "Protocols:" line with a clearer format like "Socket LB Protocols: "? Personally I was confused whether socket LB was enabled or not while debugging an issue, which is why I filed #14927. It doesn't make sense to me that you'd list a bunch of the feature aspects of the kube-proxy replacement explicitly but then for socket-lb you have to infer whether it's enabled or not based on this other line called "Protocols".

Reply:
I think the `Protocols:           TCP, UDP` line needs rework perhaps. For example we might still handle TCP, UDP when KPR is disabled and Cilium only xlated ClusterIP. So there should be some sort of indicator what is handled where. E.g. if we have partial and for sock LB we only have TCP xlation enabled, then cilium status --verbose should probably tell which is handled where. Maybe this somehow begs for an output like ...

```
Protocols:           
- North-South:      TCP, UDP
- East-West:        TCP, UDP
 - Syscall:         TCP
 - NAT:             UDP
```

... not sure if this can be made nicer / less complicated though.

Reply:
The N-S and E-W terminology is not used in the k8s context, so it might confuse users. For now I'd just do as @joestringer suggested above.

Reply:
Is that true? Some other components like traefik have been discussing east-west and north-south in k8s context:

https://traefik.io/blog/beyond-kubernetes-bringing-microservices-together-with-service-mesh/

Also, I think that anyone trying to interpret this status section probably needs the kube-proxy replacement docs anyway so we can always write a very short sentence explaining north/south and east/west. 

:man_shrugging: I'm OK with either approach.

Reply:
> Is that true?

Yeah, [the source of truth](https://kubernetes.io/docs/concepts/services-networking/service/) doesn't have any single mention of N-S or E-W.